test/cases/regression.sh

```diff
@@ -40,11 +42,26 @@ run_test_case () {
 # Provision the software under test.
 /usr/libexec/osbuild-composer-test/provision.sh
 
-# Run each test case.
+# Run test cases common for all distros.
 for TEST_CASE in "${TEST_CASES[@]}"; do
     run_test_case ${TESTS_PATH}/"$TEST_CASE"
 done
 
+case "${ID}" in
+    "fedora")
```

Comment:
Can this test script be made more generic and also work on RHEL ?

Reply:
It should be possible, the problem is that the script creates a proxy for RPMrepo which works in a different way for RHEL and Fedora. Fedora repositories are hosted on AWS, but RHEL repos are hosted on PSI. I'm not sure we can make this change in time for the Z stream update.